frontend/workflows/projectCatalog/src/details/helpers.tsx

```diff
@@ -3,18 +3,46 @@ import type { clutch as IClutch } from "@clutch-sh/api";
 import { client, Grid, Link, styled, TimeAgo as EventTime, Typography } from "@clutch-sh/core";
 import { faClock } from "@fortawesome/free-regular-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import * as _ from "lodash";
```

Comment:
Are bundle sizes still ok with importing all of this, aka things are tree shaken? I know that lodash is quite a large library.

Reply:
They should be, but we currently do this in multiple places in Clutch so it won't affect anything more than it does already. It's on my list to do some more investigation and I'll adjust the imports as necessary to handle it after.